src/openforms/registrations/tasks.py

```diff
@@ -162,31 +192,40 @@ def register_submission(submission_id: int) -> None:
 
     if not plugin.is_enabled:
         logger.debug("Plugin '%s' is not enabled", backend)
-        e = RegistrationFailed("Registration plugin is not enabled")
+        exc = RegistrationFailed("Registration plugin is not enabled")
         submission.save_registration_status(
-            RegistrationStatuses.failed, {"traceback": str(e)}
+            RegistrationStatuses.failed, {"traceback": str(exc)}
```

Comment:
probably should also be refactored to `traceback.format_exc()`? Though you'd have to use `traceback.format_exception(exc)` to be able to pass in the exception instance (and then you have to `"".join(...)`) it